### DISCUSSION TITLE
We need an avatar for this organization

### DISCUSSION THREAD

**Discussion by oood** [abbrcode/abbreviations-in-code#89] [Ideas]
Currently, our organization (https://github.com/abbrcodes) does not have an avatar or logo.

We may need it.

**Two options:**

- look for it on the internet, or 
- we make it ourselves

But to be honest, I don't know what logo should be used for this project, maybe AC? (sounds weird, lol).

**The point is:**

- simple
- code
- efficient

**Copyright-free images can be found at the following sites:**

https://unsplash.com/

https://www.flickr.com/

**Comment by T1xx1** [comment]
I can try to do it on [canva](https://canva.com)
Any suggestions?

**Reply by oood** [reply]
> I can try to do it on [canva](https://canva.com) Any suggestions?

No specific ideas, maybe tomorrow there will be inspiration, don't you need to play games?

**Reply by oood** [reply]
I just record some ideas and link them to [Brevis](https://github.com/orgs/abbrcodes/projects/1).

So, it doesn't mean we need to implement it right away, lol, feel free to do it anytime.

**Comment by oood** [comment]
Maybe a cup is enough? Simple and code related. 

(because I'm pretty sure most people have a cup next to their computer for tea or coffee)


![cup](https://user-images.githubusercontent.com/101416068/223557409-fad99c4f-28f1-413c-8f38-b444ec08ae4c.jpg)


Image from: https://unsplash.com/photos/nDd3dIkkOLo

**Reply by T1xx1** [reply]
For real idk.

**Reply by oood** [reply]
lol

**Comment by philipeachille** [comment]
As my vote for naming is  "abbrcode" I'd use an image of an apricot

**Reply by T1xx1** [reply]
What, why? Hahahah

**Comment by T1xx1** [comment]
![Abbreviations](https://user-images.githubusercontent.com/69322987/224440740-2c64ab07-57ed-4af9-9280-3c8d6a1ba16a.png)

**Reply by oood** [reply]
love it!

**Reply by oood** [reply]
I love the color, very approachable.

**Reply by T1xx1** [reply]
Thanks, I'm setting it up as I can.

**Comment by T1xx1** [comment]
Org avatar set!
Set done [task](https://github.com/orgs/abbrcode/projects/7/views/2?pane=issue&itemId=22319836)

**Reply by oood** [reply]
This looks amazing!
